Apps/W1/ShopifyConnector/app/Products/Codeunits/ShpfyProductPriceCalc.Codeunit.al

```diff
@@ -0,0 +1,180 @@
+/// <summary>
+/// Codeunit Shpfy Product Price Calc. (ID 30182).
+/// </summary>
+codeunit 30182 "Shpfy Product Price Calc."
+{
+    Access = Internal;
+    EventSubscriberInstance = Manual;
+    SingleInstance = true;
+
```

Comment:
This is correct, because the permissions are already in the role of "D365 BASIC"
Previous we use the permission sets in an xml file, because the permission objects didn't exists.
With the XML file, we must also included these permissions, but now with the permission set objects is this not necessary anymore.

Reply:
Updated with commit: "Inderect Permissions"